nilearn/datasets/atlas.py

```diff
@@ -808,7 +821,8 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
     return Bunch(**params)
 
 
-def fetch_atlas_allen_2011(data_dir=None, url=None, resume=True, verbose=1):
+def fetch_atlas_allen_2011(data_dir=None,
+                               verbose=1):
```

Comment:
Good practice is to not do chances in a PR that are cosmetic outside of the code that we modify. The reason is that it creates a cognitive overhead reviewing the PR and that it is a recipe for merge conflicts.

Reply:
I added new imports here, so figured this would be a decent point to do so.